packages/svelteui-composables/src/utilities/use-focus-return/use-focus-return.ts

```diff
@@ -0,0 +1,29 @@
+export interface useFocusReturnResult {
+	handleFocusReturn: (opened: boolean, shouldReturnFocus?: boolean) => void;
+	returnFocus: () => void;
+}
+
+/** Return utilities for focus last active element, useful for Modal and Drawer */
+export function useFocusReturn(): useFocusReturnResult {
+	let lastActiveElement: HTMLElement | undefined;
+	const returnFocus = () => {
+		if (
+			lastActiveElement &&
+			'focus' in lastActiveElement &&
+			typeof lastActiveElement.focus === 'function'
+		) {
+			lastActiveElement?.focus({ preventScroll: true });
```

Comment:
No need for optional chaining, since you are already checking that it's defined in the if statement
```suggestion
			lastActiveElement.focus({ preventScroll: true });
```